source/messaging/managing-account-settings.rst

```diff
@@ -145,7 +151,10 @@ By default, you receive notifications when someone posts a message that contains
 Reply Notifications
```

Comment:
This setting doesn't exist with CRT enabled

Reply:
Thanks, @esethna! I've added a note to clarify.